code/agents.py

```diff
@@ -64,13 +65,30 @@ def  __init__(self, position, velocity, fish_id, environment, ann, image = None,
     self._time_since_last_ANN_update
     self.fish_index
     """
-    
+
+    def get_fish_forces(self):
```

Comment:
Don't use get in python method names. Call it something else like "calculate_fish_forces", or make it a property using the @property decorator.